src/transformStateWithClones.js

```diff
@@ -65,7 +65,34 @@
  * @return {Object[]}
  */
 function transformStateWithClones(state, transforms) {
-  // write code here
+  const result = [];
+  const newObj = { ...state };
+
+  for (const obj in transforms) {
+    if (transforms[obj].operation === 'addProperties') {
+      for (const key in transforms[obj].properties) {
+        newObj[key] = transforms[obj].properties[key];
```

Comment:
You can push the result immediately in result list instead of using newObj;

Reply:
you can use Object.assign here